dynamic.js

```diff
@@ -70,12 +77,29 @@ module.exports = function (fastify, opts, next) {
     if (basePath) swaggerObject.basePath = basePath
     if (consumes) swaggerObject.consumes = consumes
     if (produces) swaggerObject.produces = produces
-    if (definitions) swaggerObject.definitions = definitions
-    if (securityDefinitions) {
-      swaggerObject.securityDefinitions = securityDefinitions
-    }
-    if (security) {
-      swaggerObject.security = security
+    if (securityDefinitions) swaggerObject.securityDefinitions = securityDefinitions
+    if (security) swaggerObject.security = security
+    if (components) swaggerObject.components = components
+    if (servers) swaggerObject.servers = servers
+
+    if (opts && opts.addFastifySchemas) {
+      const schemas = fastify._schemas.store
+      const schemaKeys = Object.keys(schemas)
+      let schemasDst = swaggerObject.definitions
+      if (isOAS3) {
+        swaggerObject.components.schemas = definitions || {}
+        schemasDst = swaggerObject.components.schemas
+      } else {
+        swaggerObject.definitions = definitions || {}
+        schemasDst = swaggerObject.definitions
+      }
+
+      for (var schemaKey of schemaKeys) {
+        const schema = Object.assign({}, schemas[schemaKey])
+        const id = schema.$id
+        delete schema.$id
```

Comment:
Can you add a comment why this is done?

Reply:
Yeah. Added comment. 
So main idea - is to extract fastify schemas to swagger definitions if proper option will be set.